templates/addons/calico/values.yaml

```diff
@@ -1,13 +1,17 @@
 installation:
   cni:
     type: Calico
+    ipam:
+      type: Calico
   calicoNetwork:
     bgp: Disabled
     mtu: 1350
     ipPools:
     - cidr: 192.168.0.0/16
       encapsulation: VXLAN
-  registry: mcr.microsoft.com/oss
```

Comment:
working on getting this published but wanted to test in the mean time

Reply:
after some discussions we will use the upstream images so will removing the mcr references in this pr